Makefile

```diff
@@ -45,92 +45,92 @@ docker_build_%_image:
 # from Docker to podman. This is needed for local analyses; in order
 # to use these updated images, pass 'nopull' to scripts/run_analysis.sh
 #
-docker_build_%_sandbox:
+build_%_sandbox:
 	@# if TAG is 'latest', the two -t arguments are equivalent and do the same thing
 	docker build -t ${REGISTRY}/$(IMAGE_NAME) -t ${REGISTRY}/$(IMAGE_NAME):$(TAG) -f $(DOCKERFILE) $(DIR)
 	sudo buildah pull docker-daemon:${REGISTRY}/${IMAGE_NAME}:$(TAG)
 
-docker_build_analysis_image: DIR=$(PREFIX)
-docker_build_analysis_image: DOCKERFILE=$(PREFIX)/cmd/analyze/Dockerfile
-docker_build_analysis_image: IMAGE_NAME=analysis
+build_analysis_image: DIR=$(PREFIX)
```

Comment:
Regarding naming, in the criticality_score project I use PHONY targets with paths:

e.g:
`build/images`
`build/images/analysis`
`build/images/scheduler`
`build/images/python_sandbox`

I only mention this because the hierarchical naming makes it easy to figure out what the more general case is (i.e. build all would be `make build`).

Not something necessarily for this CL, but might be worth considering.

Reply:
I'll make another issue